app/src/main/res/drawable/network_niesteli_logo.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="60.000006dp"
+    android:height="59.999998dp"
+    android:viewportWidth="60.000006"
+    android:viewportHeight="59.999998">
+
+    <group
+            android:translateX="-1.1721562"
+            android:translateY="-210.77028">
```

Comment:
Could you maybe remove the group here (from the SVG and then export again). Also please use exactly `64dp` for height and width.